x-pack/legacy/plugins/actions/server/builtin_action_types/lib/http_rersponse_retry_header.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { fromNullable } from 'fp-ts/lib/Option';
+
+const DEFAULT_RETRY_AFTER: number = 60;
+export function getRetryAfterIntervalFromHeaders(headers: Record<string, string>): number {
+  return fromNullable(headers['retry-after'])
+    .map(retryAfter => parseInt(retryAfter, 10))
+    .filter(retryAfter => !isNaN(retryAfter))
+    .getOrElse(DEFAULT_RETRY_AFTER);
```

Comment:
hmmm, the way I was thinking of handling 429/retry-after, is to NOT assume a default value, but return `retry: true` from the action, which is a hint to the caller (eg, alert runner) that the action should be retried, but we weren't told when, so the caller should decide.

With the default here, I guess, it would always return 60.  I think it should return null.

Reply:
fixed this issue in both Webhooks and Slack actions. 👍